pyhomematic/devicetypes/sensors.py

```diff
@@ -209,8 +209,20 @@ def is_motion_detection_active(self, channel=None):
         return bool(self.getBinaryData("MOTION_DETECTION_ACTIVE", channel))
 
     def get_brightness(self, channel=None):
-        """ Return brightness from 0 (dark ) to 255 (bright) """
-        return int(self.getSensorData("ILLUMINATION", channel))
+        """ Return brightness from 0 (dark ) to 163830 (bright) """
```

Comment:
That is the value provided by datapoints.py .. :)

Will remove the whitespace after dark.